themes/finna2/templates/feedcontent/content.phtml

```diff
@@ -7,19 +7,23 @@
       'content/header.phtml',
       ['heading' => $this->content()->getHeading()]
   );
-  $this->jsTranslations()->addStrings([
+  $translations = $this->jsTranslations()->getJSONFromArray([
     'rss_article_not_found' => 'rss_article_not_found',
     'rss_article_channel_link' => 'rss_article_channel_link',
   ]);
 
   if ($this->modal) {
     $setupJS = <<<JS
+        VuFind.addTranslations($translations);
         $(document).ready(function() {
           finna.contentFeed.loadFeed($('#modal .feed-content'), true);
         });
         JS;
     echo $this->inlineScript(\Laminas\View\Helper\HeadScript::SCRIPT, $setupJS, 'SET');
   }
+  if (!$this->modal) {
```

Comment:
Käytä ennemmin `$this->jsTranslations()->addStrings()` ei-modaalissa, niin menee samassa kaikkien muiden käännösten kanssa.